sycl/doc/extensions/supported/sycl_ext_oneapi_bfloat16.asciidoc

```diff
@@ -327,11 +316,14 @@ The following functions are only available when `T` is `bfloat16` or `sycl::marr
 === fma
```

Comment:
The line above is no longer correct:

> The following functions are only available when `T` is `bfloat16` or `sycl::marray<bfloat16, {N}>`, where `{N}` means any positive value of `size_t` type.

I think that paragraph could just be deleted.  The part about `{N}` being a positive value of `size_t` type is unneeded because this guaranteed by the C++ synopses below.